.github/workflows/stage-test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
         if: ${{ inputs.enable-coverage }}
         run: make test_cover
       - name: Test
-        if: ${{ ! inputs.enable-coverate }}
```

Comment:
mind opening a separate PR for this specific change?

Reply:
https://github.com/pulumi/esc/pull/158